go.mod

```diff
@@ -124,13 +124,13 @@ require (
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
-	k8s.io/apiextensions-apiserver v0.23.1 // indirect
+	k8s.io/apiextensions-apiserver v0.23.0 // indirect
```

Comment:
Was this intentionally downgraded? I would expect this to have moved to v0.23.6

Reply:
It's not a direct dependency of this repository and actually I have a hard time explaining why it was set as `k8s.io/apiextensions-apiserver v0.23.1`, `go mod graph` shows

```
k8s.io/kubernetes@v1.23.6 k8s.io/apiextensions-apiserver@v0.0.0
sigs.k8s.io/controller-tools@v0.8.0 k8s.io/apiextensions-apiserver@v0.23.0
...
```

In the end it seems that `k8s.io/kubernetes` uses `v0.0.0` version which can be easily overwritten.

I've run `go get -d k8s.io/apiextensions-apiserver@v0.23.6` to bump this.

Reply:
v0.0.0 is special token which K8s substitutes staging libraries with the zero'd major current version. So for the 1.23.1 release staging libraries are tagged as v0.23.1.